2024-01-08 01:14:12.061998+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Erreur de syntaxe dans l'instruction SQL "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT * FROM MENUS"
Syntax error in SQL statement "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT * FROM MENUS"; SQL statement:
INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES 
('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')

SELECT * FROM MENUS [42000-224]
2024-01-08 01:15:05.481313+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Erreur de syntaxe dans l'instruction SQL "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT * FROM MENUS"
Syntax error in SQL statement "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT * FROM MENUS"; SQL statement:
INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES 
('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')

SELECT * FROM MENUS [42000-224]
2024-01-08 01:20:14.149069+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Erreur de syntaxe dans l'instruction SQL "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT RESTAURANTS  FROM MENUS"
Syntax error in SQL statement "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT RESTAURANTS  FROM MENUS"; SQL statement:
INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES 
('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')

SELECT RESTAURANTS  FROM MENUS [42000-224]
2024-01-08 01:23:06.249271+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Erreur de syntaxe dans l'instruction SQL "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT * FROM MENUS"
Syntax error in SQL statement "INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES \000d\000a('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')\000d\000a\000d\000a[*]SELECT * FROM MENUS"; SQL statement:
INSERT INTO MENUS (IDENTIFIANT, ENTREES, PLATS, DESSERTS) VALUES 
('menu1', 'Salade de chèvre chaud', 'Burger', 'Tarte au citron')

SELECT * FROM MENUS [42000-224]
2024-01-08 12:32:49.941761+01:00 jdbc[25]: null
org.h2.message.DbException: La connexion n'a pas été fermée et a été récupérée par le ramasse miette.
The connection was not closed by the application and is garbage collected [90018-224]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:218)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:135)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:733)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1623)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La connexion n'a pas été fermée et a été récupérée par le ramasse miette.
The connection was not closed by the application and is garbage collected [90018-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 16 more
